.github/workflows/sketcher-builder.yml

```diff
@@ -206,5 +205,5 @@ jobs:
         if: matrix.build-output == 'wasm'
         uses: actions/upload-artifact@v4
         with:
-          name: sketcher-wasm.tar.gz
+          name: sketcher_${{ matrix.os}}_${{ matrix.build-output }}.tar.gz
```

Comment:
Is it worth tagging the build with the OS when the `.tar.gz` itself is platform-independent?